OloEditor/src/Panels/InputSettingsPanel.cpp

```
@@ -258,10 +277,16 @@ namespace OloEngine
 
             if (ImGui::Button("Create") && !nameEmpty && !duplicate)
             {
+                auto oldMap = InputActionManager::GetActionMap();
                 InputAction newAction;
```

Clang (14.0.6)

[warning] variable 'newAction' is not initialized

(cppcoreguidelines-init-variables)